cmd/cleanup/main.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"time"
+
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/controller"
+	"sigs.k8s.io/sig-storage-local-static-provisioner/pkg/cleanup/deleter"
+)
+
+// Command line flags
+var (
+	kubeconfig               = flag.String("kubeconfig", "", "Absolute path to the kubeconfig file. Only required when running out of cluster.")
+	masterURL                = flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
```

Comment:
I see in other examples that it's one of kubeconfig or master: https://github.com/kubernetes-csi/external-provisioner/blob/master/cmd/csi-provisioner/csi-provisioner.go#L73.

We're also moving away from the master terminology. "kube-api-endpoint" maybe would be better? Can you also add the format of the string in the help text.

Reply:
Updated to use "kube-api-endpoint" and specified format as "URL".